### PR TITLE
Add a meaningful issue template

### PR DESCRIPTION
**Pull request by patkar**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Tests pass? | n/a
| Fixed tickets | none
| License | MIT
| Doc PR | none

#### Description

This PR adds a meaningful issue template, which defines a structure to encourage high quality issues, which are understandable for new collaborators at every time.

#### Purpose

Most of the issues in this repo have less or no description at all. Thus it is very hard to follow your daily business as an outside collaborator. And as @webmozart is already questioning most of the issues and pull requests to add a meaningful description, this is the next logical step.

### CODE REVIEW

**Comment by wachterjohannes** [comment]
+1

**Comment by danrot** [comment]
I've got the same doubts as I've already written in https://github.com/sulu-io/sulu/pull/2080

**Comment by patkar** [comment]
You mean those?

> And I wouldn't put the options ([yes|no]) into the template. Same goes for the text under description and purpose, because it is very annoying to delete them for every issue being created.

**Comment by danrot** [comment]
Exactly

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+| Q | A
+| --- | ---
+| Bug? | no
```
Comment (patkar) [inline .github/ISSUE_TEMPLATE.md]: Why `no`? Different to a PR, where yes should enforce thinking about the answer, a no in the issue enforces thinking about it is really a bug (and if it's critical) or you only want to request a feature.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+| Critical? | no
```
Comment (patkar) [inline .github/ISSUE_TEMPLATE.md]: Not sure if this should be really here. I don't want to encourage false critical issues. Maybe this should be removed and Sulu core members can set the label, and not core members can add a critical line themself if they really think it is critical and are more involved with contributions.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+| Version Used | 1.1.x
```
Comment (patkar) [inline .github/ISSUE_TEMPLATE.md]: Important info, especially from first-timers. Should be updated with new minor/major releases ;)

Not sure if issues from not released code should be written as e.g. 1.2.x-dev or the exact `branch#commit-sha`?

**Comment by patkar** [comment]
@danrot Shortened the section texts and made defaults for the checklist, like the PR template from @webmozart

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (danrot) [inline .github/ISSUE_TEMPLATE.md]: I also think that this line should be deleted. From a user perspective every tiny bug is immediately critical, because it is affecting them directly (otherwise they won't report anyway).

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (danrot) [inline .github/ISSUE_TEMPLATE.md]: The exact commit sha would definitely  be more useful. So I would say here should be a specific tag or a commit sha.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (webmozart) [inline .github/ISSUE_TEMPLATE.md]: :+1:

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+| Feature request? | no
```
Comment (webmozart) [inline .github/ISSUE_TEMPLATE.md]: What about changing this to "New feature?" ?

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+#### Actual Behavior
+
+Describe the actual behavior, which error occurs or how the system currently behaves. 
+
+#### Steps to Reproduce
+
+1. Please collect the exact steps to reproduce the bug, don't hesitate to add code examples or screenshots.
+
+#### Possible solutions
```
Comment (webmozart) [inline .github/ISSUE_TEMPLATE.md]: "Possible Solutions", to stay consistent with the other headings

File: .github/ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+If you have already ideas how to solve the issue, add them here. Otherwise remove this part.
```
Comment (webmozart) [inline .github/ISSUE_TEMPLATE.md]: "this section"

**Comment by patkar** [comment]
Updated.

File: .github/ISSUE_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+Briefly describe the behavior you expect (bug) or you want (feature request).
+
+#### Actual Behavior
+
+Describe the actual behavior, which error occurs or how the system currently behaves. 
+
+#### Steps to Reproduce
+
+1. Please collect the exact steps to reproduce the bug, don't hesitate to add code examples or screenshots.
```
Comment (webmozart) [inline .github/ISSUE_TEMPLATE.md]: Let's simplify this instruction:

"What are the steps to reproduce this bug? Please add code examples, screenshots or links to GitHub repositories that reproduce the problem."

File: .github/ISSUE_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+| Q | A
+| --- | ---
+| Bug? | no
+| Version Used | Specific tag or commit sha
+| New Feature? | no
+
+#### Expected Behavior
+
+Briefly describe the behavior you expect (bug) or you want (feature request).
```
Comment (webmozart) [inline .github/ISSUE_TEMPLATE.md]: I'd formulate this as a simple question:

"What is the behavior you expect?"

File: .github/ISSUE_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+#### Actual Behavior
+
+Describe the actual behavior, which error occurs or how the system currently behaves.
```
Comment (webmozart) [inline .github/ISSUE_TEMPLATE.md]: I'd also formulate this as a question:

"How does Sulu behave at the moment?"

**Comment by webmozart** [comment]
\o/
